app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
This is a code quality issue. According to the checklist, it's better to avoid using an unnecessary `else` statement. In this case, you can just return 0 without the `else` statement. This makes the code cleaner and easier to read.

Reply:
but I don't have `else` here...